content/docs/features/querying.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Querying'
+section: 'Features'
+priority: 3
+hidden: false
+---
+
+import { Link } from 'gatsby'
+
+## Overview
+
+Most OrderCloud resources have list endpoints that can return many thousands, even hundreds of thousands of items. Being able to effectively query this data is not only practical but necessary. We understand how important this is and that's why we've built an assortment of api tools that will help you hone in on and extract exactly the data you and your users need. Up first, let's talk about Search.
+
+## Searching
+
+Google-esque searches, simply provide a search term.
+
+This type of querying is best for natural language searches like a product search on a buyer site. Our search handles case-insensitive word stemming (plural inference for example), synonyms, and intelligently ranks results based on fields relevant to each resource. If you're curious about which default fields are taken into account and how much weight they're given for a search check out our <Link to="/api-reference">API reference</Link>.
+
+```http
+GET https://api.ordercloud.io/v1/me/products?search=bike HTTP/1.1
+```
+
+This will return all results such as bike, words including bike, bikes, bicycle, etc. The fields searched include `ID`,`Name`, and `Description`. (This of course varies from endpoint to endpoint). Fields marked "Searchable" in the API reference will be searched and weighted according to priority level.
+
+You can also optionally specify which fields (of the searchable fields available) are searched by providing a `searchOn` parameter for any field available on the resource you're searching (including xp). Here's we're searching on Name and xp.Color so if our search finds a match in either of those two fields we will get a result.
```

Comment:
Uppercase XP, also shouldn't be abbreviated if it's the first time you're using the abbreviation/acronym. Like how you have below in "Filtering on XP", you define XP as extended properties but put it in parentheses. That should be moved up to the first use of "XP" in the doc

Reply:
updated ⚡️